### PR TITLE
MLPAB-1477 - replace credentials with oidc role

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
# Purpose

Briefly describe the purpose of the change, and/or link to the JIRA ticket for context

Fixes MLPAB-1477

## Approach

- tag session name to identify actions required for oidc roles
- use new oidc role for ECR pull operations

### CODE REVIEW

File: .github/workflows/ui_test_job.yml [hunk 1]
```diff
@@ -62,12 +62,10 @@ jobs:
         if: inputs.run_against_image
         uses: aws-actions/configure-aws-credentials@v4.0.1
         with:
-          aws-access-key-id: ${{ secrets.aws_access_key_id }}
-          aws-secret-access-key: ${{ secrets.aws_secret_access_key }}
           aws-region: eu-west-1
-          role-to-assume: arn:aws:iam::311462405659:role/modernising-lpa-ci
+          role-to-assume: arn:aws:iam::311462405659:role/modernising-lpa-github-actions-ecr-pull
```
Comment (acsauk) [inline .github/workflows/ui_test_job.yml:66]: Does this role need to be created somewhere (or does it already exist?)

File: internal/app/trigger_build [hunk 2]
```diff
@@ -0,0 +1 @@
+trigger a build
```
Comment (acsauk) [inline internal/app/trigger_build:1]: Are we keeping this?

File: .github/workflows/ui_test_job.yml (hunk 1 above)
Comment (andrewpearce-digital) [reply .github/workflows/ui_test_job.yml:66]: it's created and used in this PR's runs (https://github.com/ministryofjustice/opg-org-infra/pull/2386)

File: internal/app/trigger_build (hunk 2 above)
Comment (andrewpearce-digital) [reply internal/app/trigger_build:1]: thanks, I'll remove this

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**